odoo/orm/fields_relational.py

```diff
@@ -161,6 +161,32 @@ def _internal_description_domain_raw(self, env) -> str | list:
             domain = list(domain)
         return domain or []
 
+    def filter_function(self, records, field_expr, operator, value):
+        getter = self.expression_getter(field_expr)
+        corecords = getter(records)
+
+        if operator == 'any':
+            assert isinstance(value, Domain)
+            corecords = value.filter_records(corecords)
```

Comment:
Again, this bypasses the override of `filtered_domain` in `analytic.mixin`.  This should invoke `corecords.filtered_domain(value)` instead.